### ISSUE TITLE
Missing Important Model Files

### ISSUE THREAD

**Issue by KellyGong** [promoe-opensource/promoe#1]
Hi, thank you for your great work on the MoE offloading! I really appreciate the effort you’ve put into LLM efficiency. However, I noticed that some important model files ``moe-layer-logits`` seem to be missing from the repository. Without these, it’s difficult to use the code. Could you please check and add them?

Thanks again for your time and contribution!

**Comment by molamooo** [comment]
Sorry for the delayed response. We've add weights of predict models in https://github.com/promoe-opensource/sparse-llm-cache-scripts/tree/master/moe-predict-models. You can also collect your own logits and train your own predict model.
